test/test_autograd.py

```diff
@@ -191,6 +191,35 @@ def backward(ctx, grad):
 
         self.assertIsNone(torch.autograd.grad(MyFunction.apply(x), x, allow_unused=True)[0])
 
+    def test_materialize_grads(self):
+        class MyFunction(Function):
+            @staticmethod
+            def forward(ctx, x):
+                return x
+
+            @staticmethod
+            def backward(ctx, grad):
+                return grad
```

Comment:
Same comment as for the c++. That way it prevents any change to AccumulateGrad from impacting this test.